src/tslint.ts

```diff
@@ -86,6 +86,21 @@ class Linter {
     private containsRule(rules: Lint.RuleFailure[], rule: Lint.RuleFailure) {
         return rules.some((r) => r.equals(rule));
     }
+
+    private getRulesDirectories(): Array<string> {
+        const rulesDirectories: Array<string> = [];
+
+        if (this.options.rulesDirectory) {
+            if (typeof this.options.rulesDirectory === "string") {
+                rulesDirectories.push(this.getRelativePath(this.options.rulesDirectory as string));
+            } else {
+                const relativePathDirectories = (this.options.rulesDirectory as Array<string>).map(dir => this.getRelativePath(dir));
+                rulesDirectories.push(...relativePathDirectories);
```

Comment:
wait this is strange; there's no need for `push` because you only ever call it once when this method is run. instead:

```ts
let rulesDirectories: string[] = [];

if (...) {
    if (...) {
        rulesDirectories = ...
    } else {
        rulesDirectories = ...
    }
}
```